components/server/src/ome/services/search/FullText.java

```diff
@@ -301,6 +301,7 @@ protected void initializeQuery(FullTextQuery ftQuery) {
         ftQuery
         .setProjection(ProjectionConstants.SCORE,
                 ProjectionConstants.ID);
+        ftQuery.setMaxResults(500);
```

Comment:
Do you think this could become param, that will correlate with https://github.com/openmicroscopy/openmicroscopy/blob/develop/components/tools/OmeroPy/src/omero/gateway/__init__.py#L4132 and do proper pagination?